app/templates/buyers/dashboard.html

```diff
@@ -62,7 +62,7 @@ <h2 class="govuk-heading-m">Unpublished requirements</h2>
             <form method="post" action="{{ url_for('.copy_brief', framework_slug=brief.framework.slug, lot_slug=brief.lot, brief_id=brief.id) }}">
               <input type="hidden" name="csrf_token" value="{{ csrf_token_value or csrf_token() }}" />
               {{ govukButton({
-                "text": "Make a copy",
+                "html": 'Make a copy<span class="govuk-visually-hidden"> of ' + brief.title + '</span>',
```

Comment:
Are the functional tests clever enough to ignore the visually-hidden text on the button?

Reply:
Currently running them locally, so I'll be able to update on that soon!

Reply:
Confirmed - functional tests pass locally - they'll happily find a part match on button text.